src/components/DropdownInput/DropdownInput.scss

```diff
@@ -0,0 +1,34 @@
+.DropdownInput {
+  &__select-box {
+    //Changing default arrow
+    -moz-appearance: none;
+    -webkit-appearance: none;
+    appearance: none;
+    background-color: #fff;
+    background-image: url('data:image/svg+xml;base64,PHN2ZyB3aWR0aD0iMTIiIGhlaWdodD0iOCIgdmlld0JveD0iMCAwIDEyIDgiIGZpbGw9Im5vbmUiIHhtbG5zPSJodHRwOi8vd3d3LnczLm9yZy8yMDAwL3N2ZyI+CjxwYXRoIGQ9Ik0xMC41ODk2IDEuMjk2NjlMMTEuMjkyOSAxLjk5OTk3TDYgNy4yOTI4NkwwLjcwNzEwNyAxLjk5OTk3TDEuNDEwMzkgMS4yOTY2OUw1LjY0NjgzIDUuNTIzOUw2IDUuODc2M0w2LjM1MzE3IDUuNTIzOUwxMC41ODk2IDEuMjk2NjlaIiBmaWxsPSIjM0M0MjU3IiBzdHJva2U9IiMzQzQyNTciLz4KPC9zdmc+Cgo=');
+    background-repeat: no-repeat, repeat;
```

Comment:
Instead of this, it would be good to use Icon component to give us the chevron icon as you did for arrow. This is more of a workaround using no repeat and other background properties.

Reply:
Hmm, I wasn't able to make it work with that method, there were some styling issues as select input of HTML isn't very customisable.